packages/vaadin-icon/src/vaadin-icon.js

```diff
@@ -6,14 +6,49 @@
 import { PolymerElement, html } from '@polymer/polymer/polymer-element.js';
 import { ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 import { ElementMixin } from '@vaadin/vaadin-element-mixin/vaadin-element-mixin.js';
-import { IconsetElement, getIconId } from './vaadin-iconset.js';
-import { renderSvg } from './vaadin-icon-svg.js';
+import { IconsetElement } from './vaadin-iconset.js';
+import { ensureSvgLiteral, renderSvg } from './vaadin-icon-svg.js';
 
 const DEFAULT_ICONSET = 'vaadin';
 
 /**
  * `<vaadin-icon>` is a Web Component for creating SVG icons.
  *
+ * ### Icon property
+ *
+ * The icon component is designed to be used a drop-in replacement for `<iron-icon>`.
+ * For example, you can use it with `vaadin-icons` like this:
+ *
+ * ```html
+ * <vaadin-icon icon="vaadin:angle-down"></vaadin-icon>
+ * ```
+ *
+ * Alternatively, you can also pick one of the Lumo icons:
+ *
+ * ```html
+ * <vaadin-icon icon="lumo:user"></vaadin-icon>
+ * ```
+ *
+ * ### Custom SVG icon
+ *
+ * You can pass a custom `svg` literal from Lit using [`svg`](#/elements/vaadin-icon#property-svg)
```

Comment:
```suggestion
 * Alternatively, instead of selecting an icon from an iconset by name, you can pass any custom `svg` literal using the [`svg`](#/elements/vaadin-icon#property-svg)
```